data/moves.ts

```diff
@@ -15586,9 +15586,10 @@ export const Moves: import('../sim/dex-moves').MoveDataTable = {
 		priority: 0,
 		flags: {protect: 1, mirror: 1, dance: 1, metronome: 1},
 		onModifyType(move, pokemon) {
-			let type = pokemon.getTypes()[0];
-			if (type === "Bird") type = "???";
-			if (type === "Stellar") type = pokemon.getTypes(false, true)[0];
```

Comment:
Out of interest, what happened to Stellar?

Reply:
`getTypes` can never return `Stellar` as a type so it's an unnecessary check.